pkg/api/v1/server_service.go

```diff
@@ -382,3 +387,28 @@ func (c *Client) ListServerCredentialTypes(ctx context.Context, params *Paginati
 func (c *Client) CreateServerCredentialType(ctx context.Context, sType *ServerCredentialType) (*ServerResponse, error) {
 	return c.post(ctx, serverCredentialTypeEndpoint, sType)
 }
+
+// BatchBomsUpload will attempt to write multiple boms to database.
+func (c *Client) BatchBomsUpload(ctx context.Context, boms []Bom) (*ServerResponse, error) {
```

Comment:
can we call this method `BillOfMaterialsBatchUpload`, so we have clear naming for exposed methods and endpoints :)